core/config.js

```diff
@@ -0,0 +1,25 @@
+Nuxeo = Nuxeo || {};
+Nuxeo.UI = Nuxeo.UI || {};
+Nuxeo.UI.config = Nuxeo.UI.config || {};
+
+Object.assign(Nuxeo.UI.config, {
+  get(path) {
```

Comment:
perhaps it would be nice to allow this to take a default value so we avoid the `||` everywhere, no ?

Reply:
This could also allow us for instance to properly handle booleans... ie: if default value given is a boolean and property value is a String we handle the String comparison here

Reply:
I agree with the fallback, but am a bit worried about the implicit type conversions. Should we always transform what we read from the configuration service as a string if it is not an object?

Reply:
My suggestion was to handle the conversions when the types of the default value and the one in the config property don't match. If you don't provide a default value or some sort of type we don't have any good way to guess it

Reply:
I find it a bit unpredictable that config.get('abc') returns a string but config.get('abc', 123) will try to return a number and might even blow up if the value is an object or a string. Should we try to handle errors and skip this if it fails to convert?